styles/_govuk-elements.scss

```diff
@@ -0,0 +1,30 @@
+// GOV.UK elements
+@import "govuk-elements-sass/public/sass/elements/helpers";                       // Helper functions and classes
```

Comment:
FYI: I was getting a `Media query expression must begin with '(' ` which is extremely misleading